tests/integ/modin/test_qcut.py

```diff
@@ -64,19 +67,18 @@ def test_qcut_series(n, q, expected_query_count):
         high_count_reason="Bug in quantile, TODO SNOW-1229442.",
     ):
         ans = pd.qcut(snow_series, q, labels=False, duplicates="drop")
-
```

Comment:
moved `assert_...` into `with`, therefore query count is doubling due to the `to_pandas` being called there.

Reply:
do we no longer need to use the error tolerance from `assert_almost_equal`?

Reply:
It passed for me locally without the error tolerance, so I think we're fine!